backend/src/impl/default_controllers_impl.py

```diff
@@ -185,12 +185,19 @@ def benchmark_benchmark_idby_creator_get(
     else:
         view_dict = json.load(f)["system"]
     plot_dict = BenchmarkUtils.generate_plots(benchmark_id)
-    views = [
-        BenchmarkUtils.dataframe_to_table(
-            k, pd.DataFrame.from_dict(v), by_creator, plot_dict
+    views = []
+    for k, v in view_dict.items():
+        if by_creator:
+            col_name = "creator"
+        elif k == "Most-underserved Languages":
```

Comment:
I think this is a bit too specific to be in the code here. Could we rather add it to the JSON configuration file, e..g `"col_name": "source_language"`? Then the code here would look and see if `col_name` is defined in the JSON configuration file, and then if it not do the if/else statement for `by_creator`.

Reply:
I tried to implement this, but it seemed to be a lot messier than what we have currently. It would require us to change generate_view_dataframe in benchmark_utils to pass in the col_name along with the view_name and the view_df. However, the original_df is handled a little differently, so we would need to handle it separately. The issue with implementing this entirely within default_controllers_impl.py is that we don't have a good way to access the views directly from the config file by name.